corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,34 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+        $("#conditional-alert-save-btn").click(function () {
```

Comment:
Since this page is already controlled by knockout, it's better not to mix in jQuery-heavy code - it's confusing to have to reason about both frameworks at once. However, it looks like this page uses a different knockout model for each panel (Basic Info, Rule Criteria, Schedule & Recipients), and those are all contained in a larger form, so it might not be straightforward to apply knockout to that form. @mkangia what do you think makes the most sense here?

Reply:
Sorry, missed this.

Thinking about this a little more, why not just load the count of cases in django and have it available on the web page using initial_page_data or probably just embed in the modal. Seems like we are only concerned with 'edit' and hence the case type is already known so we could just run `CaseES().domain(domain).case_type(case_type).count()`.
The cost of running this each time the edit form is opened seems to be okay to avoid making the web end complex.

If that can't be done for any reason,
I agree that the Conditional alert forms are really complex. If @adil-uddin can try doing this the knockout way once, and share how easy/hard it would be, that would be useful to weigh in.